modules/core/src/constants/profile-constants.ts

```diff
@@ -33,8 +33,7 @@ export class ProfileConstants {
     // SCIM2 schema IDs
     public static readonly SCIM2_CORE_USER_SCHEMA: string = "urn:ietf:params:scim:schemas:core:2.0:User";
     public static readonly SCIM2_ENT_USER_SCHEMA: string = "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User";
-    public static readonly SCIM2_WSO2_USER_SCHEMA: string = "urn:scim:wso2:schema:User";
-    public static readonly SCIM2_WSO2_CUSTOM_SCHEMA: string = "urn:scim:wso2:schema";
+    public static readonly SCIM2_WSO2_USER_SCHEMA: string = "urn:scim:wso2:schema";
```

Comment:
wouldn't this affect any existing functionality?

Reply:
Those claims were added by my previous PR and ```SCIM2_WSO2_CUSTOM_SCHEMA``` is redundant and we can contains the dialect value ```urn:scim:wso2:schema``` in one single constant.  So this modification will not affect any existing functionality.